### ISSUE TITLE
Javascript intellisence is not correct upon hovering

### ISSUE THREAD

**Issue by Siddharth48** [Microsoft/vscode#31705]
- VSCode Version: Code 1.14.2 (cb82febafda0c8c199b9201ad274e25d9a76874e, 2017-07-19T23:34:09.706Z)
- OS Version: Windows_NT ia32 10.0.15063

When I hover over a method or function, it shows he info that is not correct.
For example, in the given part of node.js code
```
form.parse(req, function (formerror, fields, files) {
```
When I hovered over the function paramaters in some of the previous versions of vs code, they showed them more detailed and correct like this on hovering
```
formerror(paramater) : error /* I am sorry if my syntax of displaying the hover is incorrect*/
fields(paramater) : Fields
files(paramater) : Files
```
But in this update and in the previous one, it only shows "any" as type on hovering. I have even updated the typescript version by changing the typescript.tsdk setting as shown in the screenshot
![screenshot](https://user-images.githubusercontent.com/26246421/28743762-9d836250-746f-11e7-8037-30f637290dd7.png)

**Comment by mjbvz** [comment]
What is the signature of `form.parse`? We'll only be able to infer the types of the inner function if `form.parse` is correctly typed.

Also can you please provide a minimal code example that demonstrates this issue

**Comment by Siddharth48** [comment]
```
var form = new formidable.IncomingForm()
    form.parse(req, function (formerror, fields, files) {
    var firstname = fields.firstname
    var lastname = fields.lastname
    var email = fields.email
    var password = fields.password
```
Say for example if you hover parse, then it will only show 'any' instead of its sytanx which vs code displayed in the previous version which was like ```'(function) parse: [request, function (Error, Fields, Files)]```. But now it is only  displaying 'any'. I am sorry if my syntax of displaying the hover in not right

**Comment by mjbvz** [comment]
How are you importing `formidable`? After adding `const formidable = require('formidable')` and see this hover info:

![screen shot 2017-08-01 at 5 31 18 pm](https://user-images.githubusercontent.com/12821956/28852610-54c0239c-76df-11e7-9c24-2a6cff84d612.png)

The signature is coming from VS Code's [automatic type acquisition](https://code.visualstudio.com/Docs/languages/javascript#_automatic-type-acquisition)

**Comment by Siddharth48** [comment]
I ultimately found the solution.
I made a jsconfig.json file with the following data and kept it in the folder where my project.js file was located
```
"typeAcquisition": {
    "include": [
        "formidable",
    ]
}
```
After adding this file, VS Code showed the correct Signature for the parse and other methods upon hovering. This feature is what I found the best that it shows the method's type and other properties when hovered. I am sorry disturbing you
